src/Game.jsx

```diff
@@ -1,24 +1,35 @@
+import { useState } from "react";
+import GameItem from "./GameItem";
+
 export default function Game({ gameSectionText, gameCategories, games }) {
+  const [category, setCategory] = useState("All Games");
```

Comment:
@rakshyaaa It would be better if we stored the `idx` as selected rather than the category in the state to make it more generic.

```
const [idx, setSelectedIdx] = useState(0);
```